src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs

```diff
@@ -745,11 +748,14 @@ private class SplitElementResult
             // Determine the 'Display Address' which is the address we display to the user.
             // This is mainly determined by the 'display == ??' value.
             CalculateDisplayAddress(addressElement, listing.Address);
-            
+
             // Technically, the <municipality/> element is not a child of the <address/> element.
             // But I feel that it's sensible to still parse for it, in here.
             document.ValueOrDefaultIfExists(municipality => listing.Address.Municipality = municipality, "municipality");
 
+            // Street View
+            var streetView = addressElement.ValueOrDefault("streetView");
+

```

Comment:
why this?

Reply:
that was on the parse list but was removed, will remove